bemserver_ui/static/scripts/modules/views/services/cleanupManage.js

```diff
@@ -23,15 +28,19 @@ class ServiceCleanuManageView {
 
         this.#initEventListeners();
     }
-
+    
     #cacheDOM() {
         this.#messagesElmt = document.getElementById("messages");
 
-        this.#stateOnRadioElmt = document.getElementById("svc_state_on");
-        this.#stateOffRadioElmt = document.getElementById("svc_state_off");
+        this.#stateOnRadioElmt = document.getElementById("svc-state-on");
```

Comment:
Those HTML elements ids do not need to be changed. So this update is not needed.